src/Panel.vue

```diff
@@ -50,6 +50,7 @@
             </div>
         </div>
     </span>
+    <hr class="dashed-line" v-show = "this.type == 'seamless'"/>
```

Comment:
Move this right after `<div class="panel-body">...</div>` so it's only shown when expanded:

```html
<hr class="m-0" v-show="this.type === 'seamless'" />
```

Add this from Bootstrap 4's [Spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/) utility:

```css
.m-0 {
    margin: 0 !important;
}
```